docs/source/developers/python.rst

```diff
@@ -563,19 +563,23 @@ Then run the unit tests with:
    the Python extension. This is recommended for development as it allows the
    C++ libraries to be re-built separately.
 
-   If you want to bundle the Arrow C++ libraries with ``pyarrow``, add
-   the ``--bundle-arrow-cpp`` option when building:
+   If you are using the conda package manager then conda will ensure Arrow C++
+   libraries are found. In case you are *not* using conda then you have to
```

Comment:
Style nits :-)
```suggestion
   If you are using the conda package manager then conda will ensure the Arrow C++
   libraries are found. In case you are *not* using conda then you have to:
```